find/finder.go

```diff
@@ -312,6 +318,7 @@ func (f *Finder) DatastoreList(ctx context.Context, path string) ([]*object.Data
 		if ref.Type == "Datastore" {
 			ds := object.NewDatastore(f.client, ref)
 			ds.InventoryPath = e.Path
+			ds.DatacenterPath = f.dc.InventoryPath
```

Comment:
@dougm In one of my projects I'm using absolute path to the datastore, e.g. `/MyDatacenter/datastore/vm-storage01` and use a `Finder.Datastore()` to get the datastore object, but now I get panics because of this.

What would be the recommended way without having to call `SetDatacenter()`, because that would be another parameter I need to add to my project and that's already lots of changes I need to do to adjust the code to the recent change?

Thanks,
Marin